Docker build is broken

Hi all,

This PR broke `docker build`: #697 

Related to that PR, note that `make init js` is called already in the `Dockerfile` script, so calling it again from `setup.py` seems superfluous, and in the docker image it probably no longer has access to `npm` at that stage.

Output log:
```
 => ERROR [venvbuilder 4/4] RUN python3 -m venv /isso  && . /isso/bin/activate  && pip3 install --no-cache-dir --upgrade pip  && p  8.7s
------
 > [venvbuilder 4/4] RUN python3 -m venv /isso  && . /isso/bin/activate  && pip3 install --no-cache-dir --upgrade pip  && pip3 install --no-cache-dir gunicorn flask  && python setup.py install  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*:
#16 3.805 Collecting pip
#16 4.007   Downloading pip-21.3.1-py3-none-any.whl (1.7 MB)
#16 4.284 Installing collected packages: pip
#16 4.284   Attempting uninstall: pip
#16 4.285     Found existing installation: pip 20.2.1
#16 4.419     Uninstalling pip-20.2.1:
#16 4.427       Successfully uninstalled pip-20.2.1
#16 5.349 Successfully installed pip-21.3.1
#16 6.198 Collecting gunicorn
#16 6.400   Downloading gunicorn-20.1.0-py3-none-any.whl (79 kB)
#16 6.534 Collecting flask
#16 6.582   Downloading Flask-2.0.2-py3-none-any.whl (95 kB)
#16 6.609 Requirement already satisfied: setuptools>=3.0 in /isso/lib/python3.8/site-packages (from gunicorn) (49.2.1)
#16 6.689 Collecting click>=7.1.2
#16 6.738   Downloading click-8.0.3-py3-none-any.whl (97 kB)
#16 6.853 Collecting Werkzeug>=2.0
#16 6.902   Downloading Werkzeug-2.0.2-py3-none-any.whl (288 kB)
#16 7.009 Collecting itsdangerous>=2.0
#16 7.059   Downloading itsdangerous-2.0.1-py3-none-any.whl (18 kB)
#16 7.133 Collecting Jinja2>=3.0
#16 7.181   Downloading Jinja2-3.0.3-py3-none-any.whl (133 kB)
#16 7.363 Collecting MarkupSafe>=2.0
#16 7.413   Downloading MarkupSafe-2.0.1-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (30 kB)
#16 7.482 Installing collected packages: MarkupSafe, Werkzeug, Jinja2, itsdangerous, click, gunicorn, flask
#16 7.945 Successfully installed Jinja2-3.0.3 MarkupSafe-2.0.1 Werkzeug-2.0.2 click-8.0.3 flask-2.0.2 gunicorn-20.1.0 itsdangerous-2.0.1
#16 8.494 running install
#16 8.545 running bdist_egg
#16 8.546 running egg_info
#16 8.546 creating isso.egg-info
#16 8.547 writing isso.egg-info/PKG-INFO
#16 8.547 writing dependency_links to isso.egg-info/dependency_links.txt
#16 8.547 writing entry points to isso.egg-info/entry_points.txt
#16 8.547 writing requirements to isso.egg-info/requires.txt
#16 8.548 writing top-level names to isso.egg-info/top_level.txt
#16 8.548 writing manifest file 'isso.egg-info/SOURCES.txt'
#16 8.550 reading manifest file 'isso.egg-info/SOURCES.txt'
#16 8.551 reading manifest template 'MANIFEST.in'
#16 8.551 warning: no files found matching 'man/man1/isso.1'
#16 8.551 warning: no files found matching 'man/man5/isso.conf.5'
#16 8.552 writing manifest file 'isso.egg-info/SOURCES.txt'
#16 8.552 installing library code to build/bdist.linux-x86_64/egg
#16 8.552 running install_lib
#16 8.552 running build_py
#16 8.567 npm install
#16 8.568 make: npm: Command not found
#16 8.568 make: *** [Makefile:40: init] Error 127
#16 8.570 Traceback (most recent call last):
#16 8.570   File "setup.py", line 23, in <module>
#16 8.570     setup(
#16 8.570   File "/isso/lib/python3.8/site-packages/setuptools/__init__.py", line 165, in setup
#16 8.570     return distutils.core.setup(**attrs)
#16 8.570   File "/usr/local/lib/python3.8/distutils/core.py", line 148, in setup
#16 8.570     dist.run_commands()
#16 8.570   File "/usr/local/lib/python3.8/distutils/dist.py", line 966, in run_commands
#16 8.570     self.run_command(cmd)
#16 8.570   File "/usr/local/lib/python3.8/distutils/dist.py", line 985, in run_command
#16 8.570     cmd_obj.run()
#16 8.570   File "/isso/lib/python3.8/site-packages/setuptools/command/install.py", line 67, in run
#16 8.570     self.do_egg_install()
#16 8.570   File "/isso/lib/python3.8/site-packages/setuptools/command/install.py", line 109, in do_egg_install
#16 8.570     self.run_command('bdist_egg')
#16 8.570   File "/usr/local/lib/python3.8/distutils/cmd.py", line 313, in run_command
#16 8.570     self.distribution.run_command(command)
#16 8.570   File "/usr/local/lib/python3.8/distutils/dist.py", line 985, in run_command
#16 8.570     cmd_obj.run()
#16 8.570   File "/isso/lib/python3.8/site-packages/setuptools/command/bdist_egg.py", line 174, in run
#16 8.570     cmd = self.call_command('install_lib', warn_dir=0)
#16 8.570   File "/isso/lib/python3.8/site-packages/setuptools/command/bdist_egg.py", line 160, in call_command
#16 8.570     self.run_command(cmdname)
#16 8.570   File "/usr/local/lib/python3.8/distutils/cmd.py", line 313, in run_command
#16 8.570     self.distribution.run_command(command)
#16 8.570   File "/usr/local/lib/python3.8/distutils/dist.py", line 985, in run_command
#16 8.570     cmd_obj.run()
#16 8.570   File "/isso/lib/python3.8/site-packages/setuptools/command/install_lib.py", line 11, in run
#16 8.570     self.build()
#16 8.570   File "/usr/local/lib/python3.8/distutils/command/install_lib.py", line 105, in build
#16 8.570     self.run_command('build_py')
#16 8.570   File "/usr/local/lib/python3.8/distutils/cmd.py", line 313, in run_command
#16 8.570     self.distribution.run_command(command)
#16 8.570   File "/usr/local/lib/python3.8/distutils/dist.py", line 985, in run_command
#16 8.570     cmd_obj.run()
#16 8.570   File "setup.py", line 19, in run
#16 8.570     subprocess.check_call(['make', 'init', 'js'])
#16 8.570   File "/usr/local/lib/python3.8/subprocess.py", line 364, in check_call
#16 8.570     raise CalledProcessError(retcode, cmd)
#16 8.570 subprocess.CalledProcessError: Command '['make', 'init', 'js']' returned non-zero exit status 2.
------
executor failed running [/bin/sh -c python3 -m venv /isso  && . /isso/bin/activate  && pip3 install --no-cache-dir --upgrade pip  && pip3 install --no-cache-dir gunicorn flask  && python setup.py install  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*]: exit code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -7,6 +7,7 @@ RUN make init js
 # Second, create virtualenv
 FROM python:3.8-buster AS venvBuilder
 WORKDIR /src/
+ENV TOX_ENV=1
 COPY --from=jsBuilder /src .
 RUN python3 -m venv /isso \
  && . /isso/bin/activate \

